### PR TITLE
Add the doc dataset_auto_scaling in Chinese version

### PR DESCRIPTION
**Pull request by littletiger123**
### Ⅰ. Describe what this PR does
This PR adds the doc of the dataset auto-scaling in the Chinese version with a picture.  Also, the doc named monitoring.MD linked by it has the properties that should have been deleted. 

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: docs/zh/samples/dataset_auto_scaling.md [hunk 1]
```diff
@@ -0,0 +1,427 @@
+# 示例 - Dataset自定义弹性扩缩容
```
Comment (cheyang) [inline docs/zh/samples/dataset_auto_scaling.md:1]: Please move to `operation` directory. Thanks.

**Review by cheyang — CHANGES_REQUESTED** [review]

File: docs/zh/samples/dataset_auto_scaling.md [hunk 2]
```diff
@@ -0,0 +1,427 @@
...
+NAME             ENDPOINTS        AGE
+prometheus-svc   10.76.0.2:9090   6m49s
+$ kubectl get svc -n kube-system prometheus-svc
+NAME             TYPE       CLUSTER-IP      EXTERNAL-IP   PORT(S)          AGE
+prometheus-svc   NodePort   172.16.135.24   <none>        9090:32114/TCP   2m7s
+```
+
+如果希望可视化监控指标，您可以安装Grafana验证监控数据，具体操作可以参考[文档](../operation/monitoring.md)
+
+![](../../media/images/dataset_auto_scaling.png)
```
Comment (xieydd) [inline docs/zh/samples/dataset_auto_scaling.md:83]: 可以把图提上来吗？

File: docs/zh/samples/dataset_auto_scaling.md [hunk 3]
```diff
@@ -0,0 +1,427 @@
...
+3. 部署或配置 Prometheus
+
+这里通过Prometheus对于AlluxioRuntime的缓存引擎暴露的 Metrics 进行收集，如果集群内无 prometheus:
+
+```shell
+$ cd fluid
+$ kubectl apply -f integration/prometheus/prometheus.yaml
+```
+
+如集群内有 prometheus,可将以下配置写到 prometheus 配置文件中:
```
Comment (xieydd) [inline docs/zh/samples/dataset_auto_scaling.md:41]: 这里的英文和中文空格不太统一，比较奇怪，可以统一不加

**Review by xieydd — CHANGES_REQUESTED** [review]

File: docs/zh/samples/dataset_auto_scaling.md (hunk 2 above)
Comment (littletiger123) [reply docs/zh/samples/dataset_auto_scaling.md:83]: 您好，请问您的意思是无法显示图片内容需要上传图片，还是需要删除82行的空行呢？如果是前者，可以看https://github.com/littletiger123/fluid/blob/data-auto-scaling/docs/zh/samples/dataset_auto_scaling.md。

Comment (littletiger123) [reply docs/zh/samples/dataset_auto_scaling.md:83]: 您好，请问您的意思是由于图片没有加载需要上传图片，还是将第82行的空格删除呢？如果是前者，可以看一下https://github.com/littletiger123/fluid/blob/data-auto-scaling/docs/zh/samples/dataset_auto_scaling.md

Comment (xieydd) [reply docs/zh/samples/dataset_auto_scaling.md:83]: Sorry, not notice the png is upload yet.

**Review by xieydd — APPROVED** [review]
/lgtm

File: docs/zh/operation/dataset_auto_scaling.md [hunk 4]
```diff
@@ -0,0 +1,427 @@
+# 示例 - Dataset自定义弹性扩缩容
+Fluid可以通过创建Dataset对象将数据分散到Kubernetes计算节点中，作为数据交换的介质，可以有效避免了数据的远程写入和读取，提升了数据使用的效率。
```
Comment (jasonliu747) [inline docs/zh/operation/dataset_auto_scaling.md:2]: Can you put space between Chinese an English?

File: docs/zh/operation/dataset_auto_scaling.md [hunk 5]
```diff
@@ -0,0 +1,427 @@
...
+但是这里的问题是数据缓存的资源预估和预留。由于在数据生产消费之前，精准的数据预估是比较难以满足的，使用按需扩缩容对于使用者更加友好。
+按需扩缩容技术类似于page cache，对于用户而言这一层是透明的但是它带来的加速效果是很明显的。
+
+通过自定义HPA机制，使得Fluid引入了缓存弹性伸缩能力。弹性伸缩的条件就是当已有缓存数据量达到一定比例时就会发生弹性扩容缓存空间。例如
+触发条件设置为占比超过80%，总的缓存空间为10G,当数据占满到8G缓存空间时，就会触发扩缩容。
```
Comment (jasonliu747) [inline docs/zh/operation/dataset_auto_scaling.md:7]: Space between Chinese and number.

File: docs/zh/operation/dataset_auto_scaling.md [hunk 6]
```diff
@@ -0,0 +1,427 @@
...
+
+本文将向你展示这一特性。
+
+## 前提条件
+推荐使用Kubernetes 1.18以上，因为在1.18之前，HPA是无法自定义扩缩容策略的，都是通过硬编码实现的。而在1.18后，用户可以自定义扩缩容策略的，比如可以定义一次扩容后的冷却时间。
+
+
+## 具体步骤
+
+1. 安装jq工具方便解析json，在本例子中我们使用操作系统是centos，可以通过yum安装jq
```
Comment (jasonliu747) [inline docs/zh/operation/dataset_auto_scaling.md:17]: CentOS

File: docs/zh/operation/dataset_auto_scaling.md [hunk 7]
```diff
@@ -0,0 +1,427 @@
...
+```shell
+$ git clone https://github.com/fluid-cloudnative/fluid.git
+$ cd fluid/charts
+$ kubectl create ns fluid-system
+$ helm install fluid fluid
+```
+
+3. 部署或配置Prometheus
+
+这里通过Prometheus对于AlluxioRuntime的缓存引擎暴露的Metrics进行收集，如果集群内无prometheus:
```
Comment (jasonliu747) [inline docs/zh/operation/dataset_auto_scaling.md:34]: Prometheus with a capital "P"

File: docs/zh/operation/dataset_auto_scaling.md [hunk 8]
```diff
@@ -0,0 +1,427 @@
...
+
+```shell
+$ cd fluid
+$ kubectl apply -f integration/prometheus/prometheus.yaml
+```
+
+如集群内有prometheus,可将以下配置写到prometheus配置文件中:
```
Comment (jasonliu747) [inline docs/zh/operation/dataset_auto_scaling.md:41]: Prometheus with a capital "P"

**Review by jasonliu747 — COMMENTED** [review]
Thanks for your contributions!

File: docs/zh/operation/dataset_auto_scaling.md [hunk 9]
```diff
@@ -0,0 +1,427 @@
+# 示例 - Dataset自定义弹性扩缩容
```
Comment (jasonliu747) [inline docs/zh/operation/dataset_auto_scaling.md:1]: Oops, missed the header in the last review.

Comment (littletiger123) [reply docs/zh/operation/dataset_auto_scaling.md]: The problems have been modified,  thanks for your suggestions.

**Review by jasonliu747 — APPROVED** [review]
/lgtm

**Pull request merged**